doc/modules/model_evaluation.rst

```diff
@@ -242,14 +242,14 @@ permitted and will require a wrapper to return a single metric::
     >>> # A sample toy binary classification dataset
     >>> X, y = datasets.make_classification(n_classes=2, random_state=0)
     >>> svm = LinearSVC(random_state=0)
-    >>> tp = lambda y_true, y_pred: confusion_matrix(y_true, y_pred)[0, 0]
```

Comment:
I think the point was that if you want to change this for 0.19, make it a separate PR (or just commit to master).

The multi-metric stuff won't go into 0.19 as features are frozen.